lib/PHPExif/Mapper/Exiftool.php

```diff
@@ -286,12 +286,13 @@ public function mapRawData(array $data) : array
                     }
                     break;
                 case self::GPSLATITUDE:
-                    $latitudeRef = !array_key_exists('GPS:GPSLatitudeRef', $data) ?
-                        'N' : $data['GPS:GPSLatitudeRef'][0];
                     $value = $this->extractGPSCoordinates($value);
                     if ($value === false) {
                         continue 2;
                     }
+                    $latitudeRef = !array_key_exists('GPS:GPSLatitudeRef', $data)
+                        || strlen($data['GPS:GPSLatitudeRef']) < 1 ?
```

Comment:
```suggestion
                        || $data['GPS:GPSLatitudeRef'] === '' ?
```

Same question as by @ildyria.